--- origsrc/libfm-0.1.14/src/tests/Makefile.am	2010-09-24 10:48:13.000000000 -0500
+++ src/libfm-0.1.14/src/tests/Makefile.am	2011-01-20 22:08:31.722198400 -0600
@@ -3,10 +3,10 @@ include $(top_srcdir)/Makefile.decl
 NULL=
 
 AM_CPPFLAGS = \
-	-I../ \
-	-I../base \
-	-I../job \
-	-I../gtk \
+	-I$(top_srcdir)/src \
+	-I$(top_srcdir)/src/base \
+	-I$(top_srcdir)/src/job \
+	-I$(top_srcdir)/src/gtk \
 	$(GIO_CFLAGS) \
 	$(MENU_CACHE_CFLAGS) \
 	$(DBUS_CFLAGS) \
