--- a/configure.ac	2012-09-20 16:37:08.000000000 -0500
+++ b/configure.ac	2012-10-19 00:19:10.619775800 -0500
@@ -27,7 +27,7 @@ if test x"$enable_actions" = xyes; then
     if test x"$VALAC" = x; then
         AC_ERROR([No Vala compiler found but it is required.])
     fi
-    LIBFMACTIONS='$(srcdir)/actions/libfmactions.la'
+    LIBFMACTIONS='actions/libfmactions.la'
     ACTIONS_SUBDIR='actions'
     AC_DEFINE(HAVE_ACTIONS, 1, [Define to 1 to enable custom actions])
 else
--- a/src/tests/Makefile.am	2010-09-24 10:48:13.000000000 -0500
+++ b/src/tests/Makefile.am	2011-01-20 22:08:31.722198400 -0600
@@ -3,11 +3,11 @@ include $(top_srcdir)/Makefile.decl
 NULL=
 
 AM_CPPFLAGS = \
-	-I../ \
-	-I../base \
-	-I../actions \
-	-I../job \
-	-I../gtk \
+	-I$(top_srcdir)/src \
+	-I$(top_srcdir)/src/base \
+	-I$(top_srcdir)/src/actions \
+	-I$(top_srcdir)/src/job \
+	-I$(top_srcdir)/src/gtk \
 	$(GIO_CFLAGS) \
 	$(MENU_CACHE_CFLAGS) \
 	$(DBUS_CFLAGS) \
