--- origsrc/libfm-1.2.0/src/base/fm-module.c	2014-02-18 17:59:19.000000000 -0600
+++ src/libfm-1.2.0/src/base/fm-module.c	2014-06-08 20:37:13.815915000 -0500
@@ -386,7 +386,7 @@ static gboolean _fm_modules_load(gpointe
         g_debug("scanning modules directory %s", dir_name);
         while ((file = g_dir_read_name(dir)) != NULL)
         {
-            if (!g_str_has_suffix(file, ".so")) /* ignore other files */
+            if (!g_str_has_suffix(file, "." G_MODULE_SUFFIX)) /* ignore other files */
                 continue;
             g_debug("found module file: %s", file);
             g_string_printf(str, "%s/%s", dir_name, file);
