--- origsrc/libfm-1.2.0/configure.ac	2014-02-18 18:05:14.000000000 -0600
+++ src/libfm-1.2.0/configure.ac	2014-06-08 20:30:20.880296400 -0500
@@ -42,7 +42,7 @@ AM_COND_IF(EXTRALIB_ONLY,
     [enable_actions=no])
 if test x"$enable_actions" = xyes; then
     dnl no vala compiler required when sources are from dist tarball
-    if test -f src/actions/libfmactions_la_vala.stamp; then
+    if test -f $srcdir/src/actions/libfmactions_la_vala.stamp; then
         AM_PROG_VALAC
     else
         AM_PROG_VALAC([0.13.0])
--- origsrc/libfm-1.2.0/src/modules/Makefile.am	2014-02-18 17:59:19.000000000 -0600
+++ src/libfm-1.2.0/src/modules/Makefile.am	2014-06-08 20:44:49.419974000 -0500
@@ -3,7 +3,7 @@
 NULL =
 # we use glib/gio/libfm only by default
 AM_CPPFLAGS = \
-	-I.. -I../base -I../job \
+	-I$(top_srcdir)/src -I$(top_srcdir)/src/base -I$(top_srcdir)/src/job \
 	-Werror-implicit-function-declaration \
 	$(GIO_CFLAGS)
 
@@ -39,17 +39,17 @@ install-exec-hook:
 	rm -f $(DESTDIR)$(pkglibdir)/*.a
 
 # module-specific parameters
-vfs_menu_la_CFLAGS = $(MENU_CACHE_CFLAGS) -I../extra
+vfs_menu_la_CFLAGS = $(MENU_CACHE_CFLAGS) -I$(top_srcdir)/src/extra
 vfs_menu_la_LIBADD = $(MENU_CACHE_LIBS) $(top_builddir)/src/libfm-extra.la
 
-gtk_fileprop_x_desktop_la_CFLAGS = $(GTK_CFLAGS) -I../gtk
+gtk_fileprop_x_desktop_la_CFLAGS = $(GTK_CFLAGS) -I$(top_srcdir)/src/gtk
 gtk_fileprop_x_desktop_la_LIBADD = $(GTK_LIBS)
 
-gtk_fileprop_x_shortcut_la_CFLAGS = $(GTK_CFLAGS) -I../gtk
+gtk_fileprop_x_shortcut_la_CFLAGS = $(GTK_CFLAGS) -I$(top_srcdir)/src/gtk
 gtk_fileprop_x_shortcut_la_LIBADD = $(GTK_LIBS)
 
-gtk_menu_actions_la_CFLAGS = -I../actions $(GTK_CFLAGS) -I../gtk
-gtk_menu_actions_la_LIBADD = $(GTK_LIBS)
+gtk_menu_actions_la_CFLAGS = -I$(top_srcdir)/src/actions $(GTK_CFLAGS) -I$(top_srcdir)/src/gtk
+gtk_menu_actions_la_LIBADD = $(GTK_LIBS) $(top_builddir)/src/libfm-gtk.la
 
-gtk_menu_trash_la_CFLAGS = $(GTK_CFLAGS) -I../gtk
-gtk_menu_trash_la_LIBADD = $(GTK_LIBS)
+gtk_menu_trash_la_CFLAGS = $(GTK_CFLAGS) -I$(top_srcdir)/src/gtk
+gtk_menu_trash_la_LIBADD = $(GTK_LIBS) $(top_builddir)/src/libfm-gtk.la
--- origsrc/libfm-1.2.0/src/tests/Makefile.am	2013-12-05 09:05:58.000000000 -0600
+++ src/libfm-1.2.0/src/tests/Makefile.am	2014-06-08 20:46:17.417007200 -0500
@@ -3,11 +3,11 @@ include $(top_srcdir)/Makefile.decl
 NULL=
 
 AM_CPPFLAGS = \
-	-I../ \
-	-I../base \
-	-I../extra \
-	-I../actions \
-	-I../job \
+	-I$(top_srcdir)/src/ \
+	-I$(top_srcdir)/src/base \
+	-I$(top_srcdir)/src/extra \
+	-I$(top_srcdir)/src/actions \
+	-I$(top_srcdir)/src/job \
 	$(GIO_CFLAGS) \
 	-Werror-implicit-function-declaration \
 	$(NULL)
